src/openrct2/ride/TrackDesignRepository.cpp

```diff
@@ -346,7 +346,10 @@ class TrackDesignRepository final : public ITrackDesignRepository
 public:
     static std::string GetNameFromTrackPath(const std::string &path)
     {
-        return Path::GetFileNameWithoutExtension(path);
+		std::string name = Path::GetFileNameWithoutExtension(path);
```

Comment:
Please replace the tabs with spaces. We use spaces in C++ files and tabs in C files (which is confusing, I admit, but we're working on that).